charts/tempo-distributed/templates/compactor/deployment-compactor.yaml

```
@@ -0,0 +1,61 @@
+{{- if .Values.compactor.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "tempo.compactorFullname" . }}
```

I'd suggest to be consistent and use `include` only throughout the chart.
```suggestion
  name: {{ include "tempo.compactorFullname" . }}
```